### ISSUE TITLE
Feature Request: Parse URLs out of the v3 urldefense re-written URLs

### ISSUE THREAD

**Issue by seanmcfeely** [ace-ecosystem/urlfinderlib#40]
Parse URLs out of the v3 urldefense re-written URLs.

So, `http://www.apple.com/thisdoesnotexist` out of:

```
https://urldefense.com/v3/__http://www.apple.com/thisdoesnotexist__;!!P4SdNyxKAPE!SeXkINun RiEaUE9_uiTYo1zdbvNcmZgmi3nAq0Q3z4RyPHuXVyJCsVP4rFT8pfc$
```

**Comment by seanmcfeely** [comment]
Not sure it makes sense to do this in the lib. Curious what the thought is. I solved this with already built-in ace methods, so don't _really_ need it.

**Comment by automationator** [comment]
Fixed by #41
